app/models/identity.rb

```diff
@@ -17,10 +17,10 @@ class Identity < ApplicationRecord
 
   CONSENT_EXPIRATION = 1.year
 
-  IAL_MAX = 0
-  IAL1 = 1
-  IAL2 = 2
-  IAL2_STRICT = 22
+  IAL_MAX = Idp::Constants::IAL_MAX
```

Comment:
What if we removed the constants definitions here and updated refs to `Identity::IAL1` etc to `Idp::Constants::IAL1`?

Reply:
I waffled on that, but will update to use `Constants` everywhere